src/serialization/SerializationService.ts

```diff
@@ -0,0 +1,11 @@
+export class SerializationService {
+    toData(object: any) : Buffer {
```

Comment:
We need an interface called Data in this service. Its basicly a buffer wrapper and it is already defined in hazelcast.